SignalServiceKit/src/Messages/Interactions/OWSDisappearingConfigurationUpdateInfoMessage.h

```diff
@@ -20,6 +20,8 @@ NS_ASSUME_NONNULL_BEGIN
               createdByRemoteName:(nullable NSString *)remoteName
            createdInExistingGroup:(BOOL)createdInExistingGroup;
 
+- (NSString *)previewText;
```

Comment:
In some cases we can skip a transaction.

Reply:
> It's true that sometimes we can skip using the transaction but, since we do need it sometimes we need to support having a transaction available to configure cells anyway, right?

We never need them for call/info/error messages which are rendered by "system message" cells, so I was trying to simply those cells.  But I've removed this since you don't like it.

Reply:
Oh wait, I just remembered another reason why I did it - to avoid plumbing through a transaction in the measurement methods (e.g. `cellSize`) which also needs to do this kind of configuration.